### DISCUSSION TITLE
Hide ANE-Library-Symbol in Adobe Animate

### DISCUSSION THREAD

**Discussion by dgurney75** [airsdk/Adobe-Runtime-Support#2710] [General]
Any idea how I can enable the "Hide ANE-Library-Symbols" in the Air Publish settings from Animate, or could we add it as an option to the adt.cfg. I'm trying to use In-App Messaging on iOS devices using FCM and the wiki mentions this. Let me know your thoughts.

Dan

**Comment by marchbold** [comment]
Just a note, if you enable this you will break all of distriqt's extensions and a lot of others that rely on shared libraries and common code. 

Which extension is suggesting you enable it?

**Reply by dgurney75** [reply]
It’s mentioned in your Wiki for the FCM Push notifications in order to use In-App Messaging. I’m having problems getting it to work using the Firebase admin console.

**Reply by marchbold** [reply]
Do you have a link? If it's for distriqt's extensions then the docs are incorrect, we have actively been against the usage of "hide ANE library symbols" since Adobe introduced it as a workaround for bad/lazy extensions.

**Reply by dgurney75** [reply]
https://docs.airnativeextensions.com/docs/pushnotifications/firebase/add-the-extension

![Screen Shot 2023-07-06 at 11 05 19 AM](https://github.com/airsdk/Adobe-Runtime-Support/assets/8271407/b174fea8-fca4-4fd3-abf0-db07b39d9df0)

**Reply by dgurney75** [reply]
So are you saying that Firebase Cloud Messaging should work om iOS without having to disable?

**Reply by marchbold** [reply]
Hi, so that is saying you must **disable** the option. It is disabled by default. 

You use the setting to enable the feature, what we are saying there is that if you are using the option ensure it is disabled.

So if you aren't doing anything it will be disabled.

**Comment by M4rkY4** [comment]
Hi, I just wrote my tutorial in discussions, it might help in case you want to try building via Intellij IDEA while keeping Animate CC workflow https://github.com/airsdk/Adobe-Runtime-Support/discussions/2713
